.github/workflows/dotnet-large-test.yml

```diff
@@ -19,9 +19,10 @@ jobs:
     - name: Build
       run: dotnet build --no-restore
     - name: Test
-      run: dotnet test --no-restore --verbosity normal --filter TestCategory=Large
+      run: echo $GOOGLE_APPLICATION_CREDENTIALS_FILE > $PWD/googlecreds.json && GOOGLE_APPLICATION_CREDENTIALS=$PWD/googlecreds.json && dotnet test --no-restore --verbosity normal --filter TestCategory=Large
```

Comment:
picky: I would recommend breaking this out into separate lines

run: >
  command1
  command2
  command3

Reply:
Good point. I will do this in a follow up clean up PR I need to make anyway.